kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/BARuntimeDataServiceIntegrationTest.java

```diff
@@ -98,4 +98,39 @@ public void testFindTaskAssignedAsBusinessAdmin() throws Exception {
             changeUser(TestConfig.getUsername());
         }
     }
+
+    @Test
+    public void testFindTaskAssignedAsBusinessAdminSorted() throws Exception {
+        changeUser(USER_ADMINISTRATOR);
+        KieServerAssert.assertSuccess(client.createContainer(CONTAINER_ID, new KieContainerResource(CONTAINER_ID, releaseId)));
+
+        Map<String, Object> parameters = new HashMap<String, Object>();
+        parameters.put("stringData", "waiting for signal");
+        parameters.put("personData", createPersonInstance(USER_JOHN));
+
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_USERTASK, parameters);
+        Long processInstanceId2 = processClient.startProcess(CONTAINER_ID, PROCESS_ID_USERTASK, parameters);
+
+        try {
+
+            List<TaskSummary> tasks = taskClient.findTasksAssignedAsBusinessAdministrator(USER_ADMINISTRATOR, 0, 10, "processInstanceId", true);
```

Comment:
This tests ends with error, it can't find column processInstanceId because in SQL statement is renamed `taskimpl0_.processInstanceId as col_11_0_`